### PR TITLE
new tea by Bill

### PR DESCRIPTION
**Pull request by ozbillwang**

### CODE REVIEW

File: MEETINGS.md [hunk 1]
```diff
@@ -339,6 +340,8 @@
 [Four Seasons Summer]: http://www.basilurtea.com.au/tea_collection/four_seasons/four-seasons-packet-lt-summer-tea-100g.html
 [Momo Oolong]: https://usa.lupicia.com/category/select/cid/308/pid/9383/language/en
 [Fruit Shop]: http://www.basilurshop.com.au/basilur/festive-collection-100g-lt-fruit-shop
+[Dragon well]: https://en.wikipedia.org/wiki/Longjing_tea
+[longjing]: https://en.wikipedia.org/wiki/Longjing_tea
```
Comment (rkazakov) [inline MEETINGS.md:344]: @SydOps Where are these links used?

File: COLLECTION.md [hunk 2]
```diff
@@ -6,6 +6,7 @@
 
 |  #  | Brand              | Type        | Name                        | Origin      | Flavour                     |
 |-----|--------------------|-------------|-----------------------------|-------------|-----------------------------|
+| 142 | [Jia ming]         | [Green]     | [West Lake Dragon Well]     | [China]     | Green tea                   |
```
Comment (rkazakov) [inline COLLECTION.md]: There is no `Jia ming` link.

Comment (rkazakov) [reply COLLECTION.md]: [West Lake Dragon Well] - this link is used by another tea. Please pick another name and create a link.

File: MEETINGS.md (hunk 1 above)
Comment (ozbillwang) [reply MEETINGS.md:344]: They are same thing. The second one is chinese pingyin (https://en.wikipedia.org/wiki/Pinyin)

File: COLLECTION.md (hunk 2 above)
Comment (ozbillwang) [reply COLLECTION.md]: In China, many shops don't have their own website at all, they do business via taobao (a ebay-like website), wechat, friends group, but not a website. That's a too old way for them.

Comment (ozbillwang) [reply COLLECTION.md]: The problem is the way how you organize the links. Many teas have the same name, for example, `West Lake Dragon Well`,  that's not the tea someone owned. It is a common name for that type of tea.

Comment (rkazakov) [reply COLLECTION.md]: Well, I do not see the problem in links. Where I do see the problem is adding link that does not exist like `[Jia ming]` above. Why are you adding link that does not exist?

File: MEETINGS.md (hunk 1 above)
Comment (rkazakov) [reply MEETINGS.md:344]: +[Dragon well]: https://en.wikipedia.org/wiki/Longjing_tea
+[longjing]: https://en.wikipedia.org/wiki/Longjing_tea
Why are you adding these links and not using them anywhere?

**Review by rkazakov — APPROVED** [review]

**Pull request merged**